jsettlers.logic/src/jsettlers/logic/map/grid/partition/data/PartitionDataSupplier.java

```diff
@@ -46,7 +46,7 @@ public int getAmountOf(EMaterialType materialType) {
 
 	@Override
 	public IBuildingCounts getBuildingCounts() {
-		if (buildingCounts == null) {
+		if (buildingCounts == null || true) { // Disabled caching since BuildingCounts is not updating yet.
```

Comment:
@andreas-eberle Do you want to work on improving the BuildingCounts so that they update automatically?